lib/cds_importer/entity_mapper.rb

```diff
@@ -75,7 +81,12 @@ def mutate_geographical_area_membership_node!
       convert_single_geo_area_member_to_array!
 
       xml_node['geographicalAreaMembership'] = xml_node['geographicalAreaMembership'].each_with_object([]) do |geographical_area_membership, array|
-        next unless geographical_area_membership.key?('geographicalAreaGroupSid')
+        unless geographical_area_membership.key?('geographicalAreaGroupSid')
+          message = "Skipping membership import due to missing geographical area sid\n\n#{JSON.pretty_generate(geographical_membership)}\n"
+
+          instrument_warning(message, xml_node)
```

Comment:
Is it possible to have some unit tests here to check that the right message and the right xml_node is being passed?